packages/design-system/src/styles/components/_Tooltip.scss

```diff
@@ -18,6 +19,20 @@ $tooltip-background-color-inverse: $color-background !default;
   margin: 0;
 }
 
+%trigger-underline-styles {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  text-underline-position: under;
+}
+/* stylelint-disable media-feature-value-dollar-variable */
```

Comment:
Added offset for Chrome only as it interprets `text-underline-position` differently and needs the offset to make sure the line is under any text that descends.